pkg/agent/openflow/pipeline.go

```diff
@@ -2327,11 +2327,10 @@ func (f *featureEgress) snatIPFromTunnelFlow(snatIP net.IP, mark uint32) binding
 		Action().LoadPktMarkRange(mark, snatPktMarkRange).
 		Action().LoadRegMark(ToGatewayRegMark)
 	if f.enableEgressTrafficShaping {
-		// To apply rate-limit on all traffic, instead of just the first one, remove ct_state=+new.
+		// To apply rate-limit on all traffic.
 		fb = fb.Action().GotoTable(EgressQoSTable.GetID())
 	} else {
-		fb = fb.MatchCTStateNew(true).
```

Comment:
I also didn't see this change before. I assume it's not specific to this PR. Was it just a redundant match?

Reply:
It's related to the patch. Previously we only need the first packet of a connection to be marked because SNAT applies to the whole connection. But for policy routing, we need all packets of a connection to be marked.

Reply:
Makes sense